src/ModuleBlacklist/Magerun/standalone-module-scan.php

```diff
@@ -23,6 +23,8 @@
 
 if ($found) {
 	echo 'Found possible vulnerable modules: '.print_r($found, 1);
-} else {
-	echo 'No known vulnerable modules detected.';
+	exit(1);
```

Comment:
Shouldn't this be `exit(0);` ?

Reply:
I did this on intent: if vulnerabilities have been found, print the message and exit with error code 1. Otherwise, the code will run until the end, print "No known vulnerable modules detected." and exit with the OK exit code 0.

Reply:
Ah you are right, I missed the removed lines.